lastminute.yml

```diff
@@ -320,11 +320,11 @@
       reverseproxy_server: true
       tags: reverseproxy
 
-- name: Configure the cds
+- name: Configure the cds instances (new sytle)
   hosts: cds
   roles:
-    - role: cds
-      tags: cds
+    - role: cds-next
+      tags: cds-next
```

Comment:
I don't think renaming for the role is needed?

Reply:
fixed in last commit (rebased)